packages/tradershub/src/features/cfd/modals/ButtonGroups/AddAccountButtonsGroup.tsx

```diff
@@ -12,12 +12,12 @@ type TAddAccountButtonsGroupProps = {
 const AddAccountButtonsGroup = ({ password }: TAddAccountButtonsGroupProps) => {
     const { show } = Provider.useModal();
     const { getCFDState } = Provider.useCFDContext();
-    const platform = getCFDState('platform') ?? PlatformDetails.mt5.platform;
```

Comment:
for this I'm still finding a workaround because typescript is throwing undefined error 
@hamza-deriv @sergei-deriv 
bringing this up on Monday Internal Review if issue persists

Reply:
resolved